cli-feed-v3-3.json

```diff
@@ -0,0 +1,239 @@
+{
+  "tags": {
+    "v1": {
+      "release": "1.13.0",
+      "releaseQuality": "GA",
+      "hidden": false
+    },
+    "v1-prerelease": {
+      "release": "1.13.0",
+      "releaseQuality": "Prerelease",
+      "hidden": true
+    },
+    "v2": {
+      "release": "2.60.0",
+      "releaseQuality": "GA",
+      "hidden": false
+    },
+    "v2-prerelease": {
+      "release": "2.60.0",
+      "releaseQuality": "Prerelease",
+      "hidden": true
+    },
+    "v2-preview": {
+      "release": "2.60.0",
+      "releaseQuality": "Preview",
+      "hidden": true
+    },
+    "v3-preview": {
+      "release": "3.20.0",
+      "releaseQuality": "Preview",
+      "hidden": true
+    },
+    "v3-prerelease": {
+      "release": "3.20.0",
+      "releaseQuality": "Prerelease",
+      "hidden": true
+    },
+    "v3": {
+      "release": "3.20.0",
+      "releaseQuality": "GA",
+      "hidden": false
+    }
+  },
+  "releases": {
+    "1.13.0": {
+      "cli": "https://functionscdn.azureedge.net/public/1.0.28/Azure.Functions.Cli.zip",
+      "localEntryPoint": "func.exe",
+      "templateApiZip": "https://functionscdn.azureedge.net/public/TemplatesApi/1.0.3.10182.zip",
+      "sha2": "BB4978D83CFBABAE67D4D720FEC1F1171BE0406B2147EF3FECA476C19ADD9920",
+      "runtimeInformation": {
+        "dotnet": {
+          "netframework-inproc": {
+            "displayInfo": {
+              "displayName": ".NET Framework",
+              "hidden": false,
+              "diplayVersion": "v1",
+              "targetFramework": ".NET Framework"
+            },
+            "capabilites": "inProc",
+            "sdkName": "Microsoft.NET.Sdk.Functions",
+            "sdkVersion": "1.0.29" ,
+            "itemTemplates": "https://www.nuget.org/api/v2/package/Microsoft.Azure.WebJobs.ItemTemplates/1.0.3.10324",
+            "projectTemplates": "https://www.nuget.org/api/v2/package/Microsoft.Azure.WebJobs.ProjectTemplates/1.0.3.10324",
+            "projectTemplateId": "Microsoft.AzureFunctions.ProjectTemplate.CSharp.1.x",
+            "serviceAppSettings": {
+              "FUNCTIONS_EXTENSION_VERSION": "~1"
+            },
+            "windowsSiteConfig": {
+              "netFrameworkVersion": "v4.0"
+            }
+          }
+        }
+      }
+    },
+    "2.60.0": {
+      "cli": "https://functionscdn.azureedge.net/public/2.7.3188/Azure.Functions.Cli.min.win-x86.2.7.3188.zip",
+      "sha2": "f04accdc23432695786f434835b12558e603866a5c3b6d5c398965d3dccc78e9",
+      "localEntryPoint": "func.exe",
+      "templateApiZip": "https://functionscdn.azureedge.net/public/TemplatesApi/2.1.1642.zip",
```

Comment:
Rename `templateApiZip` to  `StaticTemplates` or `templates`

Reply:
That makes sense. I tried to avoid rename changes (from the last feed) unless necessary. But I am guessing because the structure has changed, might as well rename things as appropriate. I will make these updates.